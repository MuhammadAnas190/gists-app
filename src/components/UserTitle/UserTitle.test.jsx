import { render } from '@testing-library/react';
import { UserTitle } from './UserTitle';

describe('UserTitle', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(
      <UserTitle imageSrc="https://avatars.githubusercontent.com/u/8363252?v=4" name="User Name" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
