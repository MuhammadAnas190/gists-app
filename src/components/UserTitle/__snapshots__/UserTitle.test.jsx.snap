// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserTitle should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-aXZVg cWrJRO"
  >
    <img
      class="sc-gEvEer cjclob"
      src="https://avatars.githubusercontent.com/u/8363252?v=4"
    />
    <p>
      User Name
    </p>
  </div>
</DocumentFragment>
`;
